components/salesforce_rest_api/sources/common.mjs

```diff
@@ -55,8 +94,8 @@ export default {
     setNameField(nameField) {
       this.db.set("nameField", nameField);
     },
-    processEvent() {
-      throw new Error("processEvent is not implemented");
+    processTimerEvent() {
+      throw new Error("processTimerEvent is not implemented");
```

Comment:
This object is meant to be extended, akin to an abstract class. Hence, it intentionally does not implement the `processTimerEvent` method, so that objects that extend from it have to implement the method (or throw an error if they do not).